dockerfiles/cli/scripts/pre_init.sh

```diff
@@ -23,8 +23,8 @@ pre_init() {
   WS_AGENT_IN_REPO="${WS_AGENT_IN_REPO_MODULE_NAME}/target/assembly-wsagent-server-*.tar.gz"
   WS_AGENT_ASSEMBLY="ws-agent.tar.gz"
   CHE_SCRIPTS_CONTAINER_SOURCE_DIR="/repo/dockerfiles/cli/scripts"
-  CHE_LICENSE=true
-  CHE_LICENSE_URL="https://codenvy.com/legal/fair-source/"
+  CHE_LICENSE=false
+  CHE_LICENSE_URL="https://www.eclipse.org/legal/epl-v10.html"
```

Comment:
I also was thinking about that because it is defined in base so it could be taken from the parent, but then I decided to keep it there to make it clear that it should be defined  in case of overriding. And if we will need back this it will be much easier to understand what to do. So if you don't mind I would like to keep this.